benchmarker/package-lock.json

```diff
@@ -0,0 +1,1480 @@
+{
+  "name": "benchmarker",
+  "version": "1.0.0",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {
+    "": {
+      "name": "benchmarker",
+      "version": "1.0.0",
+      "license": "MIT",
```

Comment:
This doesn't seem to match package.json for some reason.

Reply:
:thinking: weird